probe/kubernetes/kubelet_test.go

```diff
@@ -0,0 +1,73 @@
+package kubernetes_test
```

Comment:
To be honest, I don't know why and when we started doing this. @paulbellamy any idea?